src/dcm/agent/plugins/builtin/unmount_volume.py

```diff
@@ -17,24 +17,46 @@
 import dcm.agent.plugins.api.base as plugin_base
 import dcm.agent.plugins.api.utils as plugin_utils
 import dcm.agent.utils as utils
+import logging
 
+_g_logger = logging.getLogger(__name__)
+
+def _translate_ubuntu_device_id(deviceId):
+    if 'essd' in deviceId:
```

Comment:
we can probably remove this ase because it will be covered with the default case